src/scripts/main.js

```diff
@@ -1,3 +1,189 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = table.children[1];
+const tableHead = table.children[0];
+
+// sort table function
+
+const sortTable = (sortBy, order) => {
+  const index = [...tableHead.firstElementChild.children]
+    .findIndex(item => item.textContent === sortBy);
+  const rows = [...tableBody.children];
+
+  rows.sort((a, b) => {
+    let valueA = a.children[index].textContent;
+    let valueB = b.children[index].textContent;
+
+    valueB = !order ? [valueA, valueA = valueB][0] : valueB;
+
+    if (+valueA) {
+      return valueA - valueB;
+    }
+
+    if (+valueA.slice(1).split(',').join('')) {
```

Comment:
Instead of using `+` to convert a string to a number, use `Number()` or `parseInt()` with a radix of 10. And it's better to create a variable for this condition